lnbits/static/js/account.js

```diff
@@ -13,11 +13,13 @@ window.app = Vue.createApp({
         'confettiStars'
       ],
       tab: 'user',
-      passwordData: {
+      credentialsData: {
         show: false,
         oldPassword: null,
```

Comment:
there still is oldPassword

Reply:
yes, it is still possible to change passwords